src/app/feed/user-info-box/user-info-box.component.html

```diff
@@ -5,18 +5,18 @@
 	<div *ngIf="(!isUserResponseLoading)" class="wrapper">
 		<div class="image-gallery">
 			<div class="profile-banner-image">
-				<img src="{{ apiResponseUser.profile_banner_url }}/600x200"/>
+				<img class="banner" src="{{ apiResponseUser.profile_banner_url }}"/>
 			</div>
 			<div class="profile-image">
 				<img src="{{ apiResponseUser.profile_image_url }}"/>
 			</div>
 		</div>
-		<div class="description">
+		<div>
 			<div class="heading">
 				<h2>{{ apiResponseUser.name }}</h2>
 				<span>@{{ apiResponseUser.screen_name }}</span>
 			</div>
-			<div class="desc">
+			<div class="desc" *ngIf="apiResponseUser.description">
```

Comment:
Previously, when there was not description for a user-x and query was made like - from:x, then there was console error i.e. undefined value was getting accessed and feed-linker was getting called continuously, even for more than 3000 times(tested by me).